MAUI/Kanban-Board/Localization.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Localization in .NET MAUI Kanban control | Syncfusion
+description: Learn here all about Localization support in Syncfusion .NET MAUI Kanban (SfKanban) control and more.
+platform: maui
+control: Kanban
+documentation: ug
+keywords: .net maui Kanban localization, sfKanban localization in .net maui, .net maui Kanban localization support, .net maui Kanban language settings, maui Kanban language support.
+---
+
+# Localization in .NET MAUI Kanban (SfKanban)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfKanban` can be localized by adding `resource` file. 
+
+## Setting CurrentUICulture to the application
+
+Application culture can be changed by setting `CurrentUICulture.` in `App.xaml.cs` file.
+
+{% tabs %}
+{% highlight c# tabtitle="App.xaml.cs" hl_lines="1 2 9 15" %}
+
+using Syncfusion.Maui.Kanban;
+using System.Globalization;
+using System.Resources;
+
+public partial class App : Application
+{
+	public App()
+	{
+		 InitializeComponent();
```

Comment:
@EzhilarasanElangovan31 code alignment not proper, modify it

Reply:
Updated alignment